.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,12 @@
+name: Lint commit messages
+on: [pull_request]
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.sha }}"
+
```

Comment:
I noticed that for [scratch-sb1-converter](https://github.com/scratchfoundation/scratch-sb1-converter) you added "cancel-in-progress: true" here. Not sure if you intentionally omitted it or not.

Reply:
Oops! I actually didn't mean to remove that. The great thing about writing shell scripts for this kind of thing is you can make the same mistake everywhere, all at once! ;)

Reply:
Oh wait, this is for the commitlint job. Yeah, I guess that doesn't really matter :)